src/register-org/components/check-your-answers/check-your-answers.component.ts

```diff
@@ -13,19 +14,28 @@ export class CheckYourAnswersComponent extends RegisterComponent implements OnIn
   public cyaFormGroup: FormGroup;
   public regulatorType = RegulatorType;
   public regulatoryType = RegulatoryType;
+  public services: string[] = [];
 
   constructor(public readonly router: Router,
     public readonly registerOrgService: RegisterOrgService
   ) {
     super(router, registerOrgService);
+  }
+
+  public ngOnInit(): void {
+    super.ngOnInit();
 
     this.cyaFormGroup = new FormGroup({
       confirmTermsAndConditions: new FormControl(null, Validators.required)
     });
-  }
 
-  public ngOnInit(): void {
-    super.ngOnInit();
+    this.registrationData.services?.forEach((serviceKey) => {
+      const service = ORGANISATION_SERVICES.find((service) => service.key === serviceKey).value;
```

Comment:
@johnbenjamin-hmcts - is `ORGANISATION_SERVICES.find()` always guaranteed to return a truthy result?

Reply:
`ORGANISATION_SERVICES` is the master list of hard-coded services to display, and also I am checking for empty services in the `forEach` loop.